src/content/core/container-registry.mdx

```diff
@@ -68,7 +68,7 @@ This command will show you information about the current okteto context in use:
 ```
 
 You can retrieve your username, password (token), and the domain for your registry from the output.
-You can also use the environment variables `OKTETO_USERNAME`, `OKTETO_TOKEN`, `OKTETO_NAMESPACE` and `OKTETO_REGISTRY` if your script is running as part of the `deploy`commands of your Okteto Manifest.
```

Comment:
Should we change it also in the `helm` code sample below? We are referring also to `OKTETO_REGISTRY` there

Reply:
ooh thanks, I had missed it. Updated now